app/src/main/java/org/gdg/frisbee/android/api/PlusPersonDownloader.java

```diff
@@ -51,7 +51,12 @@ public Response intercept(Chain chain) throws IOException {
     }
 
     @Nullable
-    public Person getPersonSync(final String gplusId) {
+    public static Person getPersonSync(final String gplusId) {
```

Comment:
Eventually I actually want to remove static and make this private function of this class. It will only be accessed here. It requires some work.